llvm/test/CodeGen/X86/keylocker-intrinsics-fast-isel.ll

```diff
@@ -25,6 +32,17 @@ define i32 @test_encodekey128_u32(i32 %htype, <2 x i64> %key, ptr nocapture %h)
 ; CHECK-NEXT:    movups %xmm5, 64(%rsi)
 ; CHECK-NEXT:    movups %xmm6, 80(%rsi)
 ; CHECK-NEXT:    retq
+;
+; EGPR-LABEL: test_encodekey128_u32:
+; EGPR:       # %bb.0: # %entry
+; EGPR-NEXT:    encodekey128 %edi, %eax # encoding: [0x62,0xf4,0x7e,0x08,0xda,0xc7]
```

Comment:
Shouldn't the compression be applied automatically?

Reply:
rebased.